### PR TITLE
Fix outdate release command.

### PR DESCRIPTION
**Pull request by msbukal**

### CODE REVIEW

**Review by danielbeaudreau — APPROVED** [review]

File: integration_test/scripts/publish-images.sh [hunk 1]
```diff
@@ -32,16 +32,16 @@ publish_adapter () {
...
   # have to do this because docker does not allow to search through tags in local storage
   version=`docker images | grep $base_name | awk '{print $2}' | grep "\." | head -n 1`
   version=${version[0]}
   import=$(publish_adapter import $version)
   export=$(publish_adapter export $version)
   body="$import\n$export"
   echo {\"tag_name\": \"$version\",\"name\": \"$version\",\"body\": \"$body\"} > /workspace/request.json
-  responseCode=$(curl -# -XPOST -H 'Content-Type:application/json' -H 'Accept:application/json' -w "%{http_code}" --data-binary @/workspace/request.json \
-  https://api.github.com/repos/$githubUser/$githubRepo/releases?access_token=$accessToken -o response.json)  
+  responseCode=$(curl -# -XPOST -H "Authorization: Bearer $accessToken" -H 'Content-Type:application/json' -H 'Accept:application/json' -w "%{http_code}" --data-binary @/workspace/request.json \
```
Comment (danielbeaudreau) [inline integration_test/scripts/publish-images.sh:43]: might need to move the " before the $accessToken, not totally sure though

Comment (msbukal) [reply integration_test/scripts/publish-images.sh:43]: Hmm I usually do `-H "Authorization: Bearer $(gcloud auth print-access-token)"` or `?access_token=$(gcloud auth print-access-token)` so I think it should be fine?

Comment (danielbeaudreau) [reply integration_test/scripts/publish-images.sh:43]: Sg, let's give it a try

**Pull request merged**